cinema/serializers.py

```diff
@@ -1 +1,134 @@
-# write serializers here
+from rest_framework import serializers
+from cinema.models import (
+    Order,
+    Ticket,
+    CinemaHall,
+    Actor,
+    Genre,
+    Movie,
+    MovieSession
+)
+
+
+class CinemaHallSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = CinemaHall
+        fields = ("name", "rows", "seats_in_row")
+
+
+class CinemaHallDetailSerializer(serializers.ModelSerializer):
+    def get_capacity(self, obj):
+        return obj.capacity
+
+    class Meta:
+        model = CinemaHall
+        fields = ("name", "rows", "seats_in_row", "capacity")
+
+
+class ActorSerializer(serializers.ModelSerializer):
+    full_name = serializers.SerializerMethodField()
+
+    class Meta:
+        model = Actor
+        fields = ("id", "first_name", "last_name", "full_name")
+
+    def get_full_name(self, obj):
+        return f"{obj.first_name} {obj.last_name}"
+
+
+class GenreSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Genre
+        fields = "__all__"
+
+
+class MovieSessionSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = MovieSession
+        fields = ("id", "show_time", "movie", "cinema_hall")
+
+
+class MovieSessionListSerializer(serializers.ModelSerializer):
+    movie_title = serializers.CharField(source="movie.title", read_only=True)
+    cinema_hall_name = serializers.CharField(
+        source="cinema_hall.name",
+        read_only=True
+    )
+    cinema_hall_capacity = serializers.IntegerField(
+        source="cinema_hall.capacity",
+        read_only=True
+    )
+
+    class Meta:
+        model = MovieSession
+        fields = ("movie_title", "cinema_hall_name", "cinema_hall_capacity")
+
+
+class MovieSessionDetailSerializer(serializers.ModelSerializer):
+    movie = serializers.SerializerMethodField()
+    cinema_hall = serializers.SerializerMethodField()
+
+    class Meta:
+        model = MovieSession
+        fields = ("id", "show_time", "movie", "cinema_hall")
+
+    def get_movie(self, obj):
```

Comment:
Type annotation is missing for the method `get_movie`. According to the checklist, do not forget about type annotation in class methods.